test/linter/test-versions.js

```diff
@@ -147,6 +147,22 @@ function checkVersions(supportData, relPath, logger) {
   }
 }
 
+/**
+ * @param {Identifier} data
+ * @param {string} [relPath]
+ */
+function findSupport(data, logger, relPath) {
```

Comment:
1. logger param is missing in JSDoc
2. Adding `@returns {void}` would also make sense. 🤓 
3. Not sure why `relPath` is in brackets. 🤷